README.md

```diff
@@ -2,97 +2,123 @@
 
 See [Ubuntu's website](https://ubuntu.com/) for more information on Ubuntu.
 
+## Table of content
+
+<!-- TOC -->
+* [Overview](#overview)
+* [Before Starting](#before-starting)
+* [Getting Started](#getting-started)
+  * [After Running The Script](#after-running-the-script)
+* [Dual-Boot - Ubuntu 22.04 with Windows 11](#dual-boot---ubuntu-2204-with-windows-11)
+  * [Requirement](#requirement)
+  * [Preparing Windows for Ubuntu](#preparing-windows-for-ubuntu)
+    * [Shrink Windows partition](#shrink-windows-partition)
+    * [Windows Drive Encryption](#windows-drive-encryption)
+  * [Installing Ubuntu](#installing-ubuntu)
+* [Notes](#notes)
+* [Change log](#change-log)
+* [Notes](#notes-1)
+  * [Slack](#slack)
+<!-- TOC -->
+
 ## Overview
 
-This folder contains installation scripts for setting up an Ubuntu 22.04 Linux machine for developers at Belighted.
```

Comment:
I agree that "folder" is not necessarily the best word to use here. Although, "doc" does not seem right either since the document does not include the scripts, the repository does, I think it might be a better way to describe it with "repository".

Reply:
will fix